### PR TITLE
Remove dead code: SetError

### PR DESCRIPTION
**Pull request by Jeschli**

### CODE REVIEW

**Comment by rhaist** [comment]
@Jeschli could you please rebase?

**Comment by Jeschli** [comment]
done.

File: bloom.go [hunk 1]
```diff
@@ -15,10 +15,7 @@ import (
 	"math"
 )
 
-// SetError represents an error with a given message related to set operations.
-type SetError struct {
-	msg string
-}
+const magicSeed = "this-is-magical"
```
Comment (rhaist) [inline bloom.go]: `magicSeed` should be deleted as well. This is obsolete with the latest PR merge.

**Comment by rhaist** [comment]
Good to go. Thank you!

**Pull request merged**